README.md

```diff
@@ -338,20 +339,71 @@ Example:
 pylint living_documentation_generator/generator.py
 ``` 
 
-## Run unit test
+### Expected Output
+This is the console expected output example after running the tool:
+```
+************* Module main
+main.py:30:0: C0116: Missing function or method docstring (missing-function-docstring)
+
+------------------------------------------------------------------
+Your code has been rated at 9.41/10 (previous run: 8.82/10, +0.59)
+```
+
+## Run Black Tool Locally
+This project uses the Black tool for code formatting.
```

Comment:
```suggestion
This project uses the [Black](https://github.com/psf/black) tool for code formatting.
```

Reply:
Change as well as for Pylint tool: [fdafc07](https://github.com/AbsaOSS/living-doc-generator/pull/30/commits/fdafc0736596047e2a543c30bd349bbc09dd1141)